intermediate_source/forward_ad_usage.py

```diff
@@ -172,5 +172,75 @@ def jvp(ctx, gI):
                          check_backward_ad=False, check_undefined_grad=False,
                          check_batched_grad=False)
 
+######################################################################
+# Functional API
+# --------------------------------------------------------------------
+# We also offer a higher-level functional API in functorch
+# for computing Jacobian-vector products that you may find simpler to use
+# depending on your use case.
+#
+# The benefit of the functional API is that there isn't a need to understand
+# or use the lower-level dual tensor API and that you can compose it with
+# other functorch transforms (like vmap); the downside is that it offers you
+# less control.
+#
+# Note that the remainder of this tutorial will require functorch
+# (https://github.com/pytorch/functorch) to run. Please find installation
+# instructions at the specified link.
+
+import functorch as ft
```

Comment:
I think this means that I will need to add `functorch` as a dependency in https://github.com/pytorch/tutorials/pull/1761. Or you could also add it in this PR I suppose, but this file isn't being run anyway as of yet.

Reply:
Good catch. Yes, we'll need to add functorch as a dependency

Reply:
Dependency added.